addons/website_forum/data/forum_default_faq.html

```diff
@@ -121,7 +121,7 @@
         <td>accept any answer (after 1 week)</td>
     </tr><tr>
         <td class="faq-rep-item"><strong>750</strong></td>
-        <td>delete any comment</td>
+        <td>delete any comment, your biography can be viewed as tooltip</td>
```

Comment:
Its not like when you have 750 karmas, You can see the autobiographies of the other users. The thing is, If a user has 750 karmas, others can see his/her expanded biography. Still you want me to change the string ?